modules/em/src/DensityMap.cpp

```diff
@@ -1104,6 +1104,84 @@ DensityMap *DensityMap::get_cropped(const algebra::BoundingBox3D &bb) {
   return cropped_dmap.release();
 }
 
+DensityMap *DensityMap::get_cropped(Particles ps, float distance, bool inverse) {
+  
+  algebra::Vector3Ds coords;
+  algebra::Vector3D vox_cent;
+  float vox_dist; 
+  bool crop;
+  Pointer<DensityMap> cropped_map;
+
+  // Always good to have a list of coordinates
+  for (unsigned int i=0; i<ps.size(); i++){
```

Comment:
Use `++i` rather than `i++`. In principle it is faster (although I suspect a modern compiler with optimize it out anyway) but either way, it indicates you're not using the original value of `i` for something.